modules/storage-persistent-storage-overview.adoc

```diff
@@ -7,17 +7,19 @@
 [id=persistent-storage-overview_{context}]
 = Persistent storage overview
 
+Stateful applications deployed in containers require persistent storage. {microshift-short} uses a pre-provisioned storage framework called persistent volumes (PV) to allow node administrators to provision persistent storage. The data inside these volumes can exist beyond the lifecycle of an individual pod. Developers can use persistent volume claims (PVCs) to request storage requirements.
+
 ifndef::microshift[]
-Managing storage is a distinct problem from managing compute resources. {product-title} uses the Kubernetes persistent volume (PV) framework to allow cluster administrators to provision persistent storage for a cluster. Developers can use persistent volume claims (PVCs) to request PV resources without having specific knowledge of the underlying storage infrastructure.
+Managing storage is a distinct problem from managing compute resources. {product-title} uses the Kubernetes persistent volume (PV) framework to allow node administrators to provision persistent storage for a node. Developers can use persistent volume claims (PVCs) to request PV resources without having specific knowledge of the underlying storage infrastructure.
```

Comment:
An `ifndef` means "if NOT MicroShift" so you'll want to revert these back to "cluster"; this content should not be showing up in our docs, but in OCP distros.